english/soar/soar.md

```diff
@@ -3,4 +3,10 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Fly to Paris] (fly-to-paris/fly-to-paris.md)
+
+[Fly to London] (fly-to-london/fly-to-london.md)
+
+[Change that wasn't pushed] (change-not-pushed)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.